app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :users, foreign_key: :author_id, inverse_of: :likes
+  belongs_to :posts, inverse_of: :likes
```

Comment:
- [ ] Same issue as above.
       1. Like should belong to an `author` and not `users`. Take care of the singularity mentioned above.
       2. Like should belong to a `post` and not `posts`. Take care of the singularity.

     Please refactor as appropriate.